config/300-channel.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+ name: channels.pubsub.cloud.run
```

Comment:
Just jotting this here. Can you also modify the 201-addressable-resolvers-clusterrole.yaml here (or add a new one) for additional verbs, I don't think as it stands today we'll be able to create channels because it only lists: get, list and watch. But perhaps I missed something along the way? Something like this:
https://github.com/knative/eventing/blob/master/config/channels/in-memory-channel/200-channelable-manipulator-clusterrole.yaml

Reply:
thanks, forgot this. Added.